realsense2_camera/src/rs435_node.cpp

```diff
@@ -11,7 +11,7 @@ RS435Node::RS435Node(ros::NodeHandle& nodeHandle,
     // if depth_exposure and depth_exposure are not provided, use the default values from cfg file
     _pnh.param("depth_exposure", _depth_exposure, -1);
     _pnh.param("depth_preset", _depth_preset, -1);
-    _pnh.param("depth_laser_power", _depth_laser_power, -1.0);
+    //_pnh.param("depth_laser_power", _depth_laser_power, -1.0);
```

Comment:
Can we remove the commented code.

Reply:
Do we want to remove dynamic reconfigure code permanently?

Reply:
Good point, guess we can leave it in there in case we find a real fix.